### DISCUSSION TITLE
About the bug of tool uuid-parser

### DISCUSSION THREAD

**Discussion by ctyytc** [he3-app/he3-feedback#299] [Bug Report] [answered]
### Version
uuid-parser 1.0.5

### Environment
Mac v2.0.0

### Steps to reproduce
Open the tool

### What is expected?
Can work

### What is actually happening?
Can't work, nothing to see.

**Comment by lyzhang1999** [comment]
Thanks for your feedback, we have created an issue tracker and will fix it as soon as possible: https://github.com/he3-app/he3-feedback/issues/300

**Comment by zhangze-github** [comment, marked as answer]
fixed，请点击工具右上角升级一下工具
![image](https://github.com/he3-app/he3-feedback/assets/41512027/2364abc7-1c0c-4512-98bc-30671dbc9af6)
